modules/client-handler/src/test/java/org/apache/ignite/client/handler/requests/sql/ClientSqlCommonTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.client.handler.requests.sql;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.Set;
+import org.apache.ignite.internal.client.sql.AllowedQueryType;
+import org.apache.ignite.internal.sql.engine.SqlQueryType;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+
+/**
+ * Tests for {@link ClientSqlCommonTest}.
+ */
+public class ClientSqlCommonTest {
+    @ParameterizedTest
+    @EnumSource(AllowedQueryType.class)
+    void testConvertAllowedTypeToQueryType(AllowedQueryType type) {
+        Set<SqlQueryType> sqlQueryTypes = ClientSqlCommon.convertAllowedTypeToQueryType(type);
+
+        assertFalse(sqlQueryTypes.isEmpty());
+
+        sqlQueryTypes.forEach(sqlQueryType -> {
```

Comment:
we need to make sure also that none of existing SqlQueryType is missing. Assume we add a new type which for some reason doesn't fall in any of the ALLOW-ed types. We need to highlight such cases